File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.py
class Solution(object):
    def minOperations(self, s):
        c=c1=0
        for i in range(0,len(s)):
            if i%2==0 and s[i]=='1':
                c+=1
            if i%2!=0 and s[i]=='0':
                c+=1

            if i%2!=0 and s[i]=='1':
                c1+=1
            if i%2==0 and s[i]=='0':
                c1+=1

        return min(c1,c)

File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.java
class Solution {
    public int minOperations(String s) {
        int c=0;
        for(int i=0;i<s.length();i++){
            if(i%2==0 && s.charAt(i)=='1'){
                c++;
            }
            if(i%2!=0 && s.charAt(i)=='0'){
                c++;
            }
        }  
        int c1=0;
        for(int i=0;i<s.length();i++){
            if(i%2!=0 && s.charAt(i)=='1'){
                c1++;
            }
            if(i%2==0 && s.charAt(i)=='0'){
                c1++;
            }
        }   
        return Math.min(c,c1);
    }
}